pages/register.tsx

```diff
@@ -66,8 +66,8 @@ export default function Register() {
     //setting user specific initial values
     formInitialValues['id'] = user?.id || '';
     formInitialValues['preferredEmail'] = user?.preferredEmail || '';
-    formInitialValues['firstName'] = user?.firstName || '';
-    formInitialValues['lastName'] = user?.lastName || '';
+    // formInitialValues['firstName'] = user?.firstName || '';
+    // formInitialValues['lastName'] = user?.lastName || '';
```

Comment:
Do we want to remove these from the initial values?

Reply:
Google account last name is mostly just non existent so last name ends up being empty anyways. People end up always changing the first name autofill anyways but ig I can split the Google first name and put in the first word. Besides that, the registration would just be like those that created an account instead.